### DISCUSSION TITLE
Support for Direction when icons are exported.

### DISCUSSION THREAD

**Discussion by agarcia-wish** [RedMadRobot/figma-export#133] [Ideas]
I don't think this is currently supported, but I would like to ask first. The idea is to tag the icons in Figma that need to support direction for Right to Left languages and then when they are exported generate the JSON with the correct data.
<img width="245" alt="Screen Shot 2021-12-01 at 11 40 33 AM" src="https://user-images.githubusercontent.com/84881359/144275514-018110bf-0e11-45ec-90e4-8eab37119ba8.png">

**Comment by subdan** [comment]
Could you show me an example of RTL icon you want to use?

Do you want to just mirror icon or have separate icon for RTL?

Where do you want to use RTL icons: inside buttons or not? Do you know about the issues when using Direction with xcassets? https://useyourloaf.com/blog/flipping-and-localizing-image-assets/

**Reply by agarcia-wish** [reply]
Sorry, for some reason I have missed your response.

I haven't faced the issues that are mentioned in the blog so far. I am able to successfully use RTL icons. 

> Could you show me an example of RTL icon you want to use?
The idea is to have a frame in Figma with all the icons that are mirrored. So, either the designer use a keyword to indicate that the icon is mirrored, or the developer adds an attribute in the yml file to indicate that all the icons need to be marked as `RTL mirrors` icons.

>Do you want to just mirror icon or have separate icon for RTL?
I just want to mirror icons. No separate icon.

**Reply by subdan** [reply]
So, if a designer adds a keyword `RTL` in the description field of the component, FigmaExport must set `Direction` to `Right to Left, Mirrors`. Did I understand you correctly?
